package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "zx",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
`index.cjs/*.cjs` should be included to`files`

Reply:
Nice catch.